packages/create-ima-app/template/app/page/home/HomeController.js

```diff
@@ -97,22 +97,21 @@ export default class HomeController extends AbstractPageController {
 
     metaManager.setTitle(title);
 
-    metaManager.setMetaName('description', description);
-    metaManager.setMetaName(
-      'keywords',
-      'IMA.js, isomorphic application, javascript, hello world'
-    );
+    metaManager.setMetaName('description', { value: description });
+    metaManager.setMetaName('keywords', {
+      value: 'IMA.js, isomorphic application, javascript, hello world',
+    });
 
-    metaManager.setMetaName('twitter:title', title);
-    metaManager.setMetaName('twitter:description', description);
-    metaManager.setMetaName('twitter:card', 'summary');
-    metaManager.setMetaName('twitter:image', image);
-    metaManager.setMetaName('twitter:url', url);
+    metaManager.setMetaName('twitter:title', { value: title });
```

Comment:
this looks really messy to me. i think that the best solution would be to add a new method `setMetaNameValue` that would take just the `value ` param